charts/cortex-agent/values.yaml

```diff
@@ -31,6 +31,9 @@ agent:
   # Agent distribution id should be retrieved from the tenant
   distributionId: ""
 
+  # Set to "false" to inject distributionId externally, e.g. Vault-agent-injector
+  createSecret: true
```

Comment:
Let's create a new hierarchy outside of `agent` (which contains agent specific settings that aren't related to k8s), like this:
```
deploymentSecret:
  create: true
```

Reply:
Added as below

```
daemonsetSecret:
  create: true
```